src/Tizen.Applications.Common/Tizen.Applications.RPCPort/Parcel.cs

```diff
@@ -18,6 +18,135 @@
 
 namespace Tizen.Applications.RPCPort
 {
+    /// <summary>
+    /// This structure represents the time stamp.
+    /// </summary>
+    /// <since_tizen> 9 </since_tizen>
+    public class TimeStamp
+    {
+        /// <summary>
+        /// Constructor with TimeStamp.
+        /// </summary>
+        /// <exception cref="InvalidIOException">Thrown when an internal IO error occurs.</exception>
+        /// <since_tizen> 9 </since_tizen>
+        internal TimeStamp(Interop.LibRPCPort.Parcel.TimeStamp time)
+        {
+            second = time.sec;
+            nanoSecond = time.nsec;
+        }
```

Comment:
커플링은 줄일수록 좋을고 이렇게하면 외부와의 커플링을 0으로 할수 있습니다.
```suggestion
        internal TimeStamp(long second, long nanoSecond)
        {
            this.second = second;
            this.nanoSecond = nanoSecond;
        }
```